service/controller/app/v1/resource_set.go

```diff
@@ -209,10 +210,22 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		}
 	}
 
+	var tillerResource controller.Resource
+	{
+		c := tiller.Config{
+			Logger: config.Logger,
+		}
+		tillerResource, err = tiller.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
 	resources := []controller.Resource{
 		appNamespaceResource,
 		appcatalogResource,
 		clientsResource,
+		tillerResource,
```

Comment:
Put it right after `clients` so we can install tiller after helmclient are up.

Reply:
Nice 👍